examples/random_sticker/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "random_sticker"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+rand = "0.8.5"
```

Comment:
I prefer don't specify the patch version in most dependencies, so I think we need to use `0.8` instead of `0.8.5`.